src/plugins/pcr/src/InSilicoPcrTask.cpp

```diff
@@ -111,6 +111,13 @@ void InSilicoPcrTask::prepare() {
     CHECK_OP(stateInfo, );
     FindAlgorithmTaskSettings reverseSettings = getFindPatternSettings(U2Strand::Complementary);
     CHECK_OP(stateInfo, );
+    if (qgetenv("UGENE_MAX_RESULTS_FOR_PRIMERS_PER_STRAND") == "1") {
```

Comment:
This should be a value: parse it as an integer.

Reply:
The documentation should be update about this env. variable and about the default value `MAX_RESULTS_FOR_PRIMERS_PER_STRAND`

Reply:
Please, update and add a link here to docs

Reply:
https://doc.ugene.net/wiki/pages/viewpage.action?pageId=69402659